models.py

```diff
@@ -177,54 +179,218 @@ def __str__(self):
 class MaintenanceDeployer:
     """Class for deploying maintenances"""
     controller: Controller
-    maintenance_devices: Counter
+    maintenance_switches: Counter
```

Comment:
`MaintenanceDeployer` and `Scheduler` look great, but on `models.py` they're doing way more than just being models or a dataclass. [Based on some guidelines that we have specified they should be in a `managers/` package since they behave like a "Manager"](https://kytos-ng.github.io/napps/mongodb.html#napps-modules), if eventually you could move then accordingly that'd be great.

Reply:
Moved these out to a separate module.